api/src/methods/estimate_gas.rs

```diff
@@ -140,7 +140,7 @@ mod tests {
         drop(state_channel);
         state_handle.await.unwrap();
 
-        let expected_response: serde_json::Value = serde_json::from_str(r#""0x6326""#).unwrap();
+        let expected_response: serde_json::Value = serde_json::from_str(r#""0x6330""#).unwrap();
```

Comment:
nit: it goes up by 4 for the deployment, just seems suspiciously low.

Reply:
Yes, good call. I dug into it more and found that we are missing charging on a whole type of gas (storage costs). I created a new issue for it: #364 . I think we should merge this PR because it still does add a necessary cost. The storage costs will be handled in another PR.